changelog/conf-order.dd

```diff
@@ -0,0 +1,13 @@
+DMD will now prefer a `dmd.conf` or `sc.ini` in the argv0 directory over the `dmd.conf` in the home directory
+
+The sequence of the inifile lookup has been changed to this order:
+
+$(OL
+  $(LI current directory)
+  $(LI exe directory (windows))
```

Comment:
To me this looks like the exe directory is not checked at all on non-Windows platforms. That's not correct?

Reply:
Ah, right, not sure what I was thinking.